### DISCUSSION TITLE
WinterCG compat?

### DISCUSSION THREAD

**Discussion by tomByrer** [CanadaHonk/porffor#164] [Ideas]
(in response to your "Node compat?" question)
TBH I think the best scenario for Porffor is edge functions; Cloudflare Workers don't have to open a new browser window every time to run a script, & ValTown doesn't have to [spawn Deno](https://blog.val.town/blog/node-spawn-performance/) so much (I think using Node)?

[WinterCG is a smaller subset](https://wintercg.org/work) than Node/Deno/Bun compat.

**Comment by CanadaHonk** [comment]
Definitely interested in it, I have already participated in it previously ;)

**Comment by matthewp** [comment]
@CanadaHonk based on your work on https://github.com/WinterTC55/proposal-cli-api I assume you're more interested in this than https://github.com/CanadaHonk/porffor/discussions/9 now?

**Comment by CanadaHonk** [comment]
@matthewp Honestly for near future node compat in Porffor will probably happen before there is a stable WinterTC CLI API

**Comment by guest271314** [comment]
There's nothing special about WinterCG. They just compiled a list of existing Web API's under their umbrella.

**Reply by tomByrer** [reply]
I agree; it is more about easier portability for me.  If I build on CloudFlare, I want to host it somewhere else when needed.

**Reply by guest271314** [reply]
What specific API's are you using?
